flink-connector/src/test/java/com/datastrato/gravitino/flink/connector/integration/test/hive/FlinkHiveCatalogIT.java

```diff
@@ -281,6 +291,59 @@ public void testGetCatalogFromGravitino() {
         numCatalogs, tableEnv.listCatalogs().length, "The created catalog should be dropped.");
   }
 
+  @Test
+  public void testHivePartitionTable() {
+    String databaseName = "test_create_hive_partition_table_db";
+    String tableName = "test_create_hive_partition_table";
+    String comment = "test comment";
+    String key = "test key";
+    String value = "test value";
+
+    doWithSchema(
+        currentCatalog(),
+        databaseName,
+        catalog -> {
+          TableResult result =
+              sql(
+                  "CREATE TABLE %s "
+                      + "(string_type STRING COMMENT 'string_type', "
+                      + " double_type DOUBLE COMMENT 'double_type')"
+                      + " PARTITIONED BY (string_type, double_type)"
+                      + " COMMENT '%s' WITH ("
+                      + "'%s' = '%s')",
+                  tableName, comment, key, value);
+          TestUtils.assertTableResult(result, ResultKind.SUCCESS);
+
+          Table table =
```

Comment:
The test covers `create hive partition table path`, but not covers the `load hive partition table path` in flink connector, YES?

Reply:
Done